### DISCUSSION TITLE
question about how robust is appwrite

### DISCUSSION THREAD

**Discussion by issasaleh** [appwrite/appwrite#5834] [Q&A] [answered]
Hi,

I'm just wondering, can Appwrite handle 100 million Documents without any problem ??
my database will have more than 100 million Documents.

thanks,

**Comment by eldadfux** [comment, marked as answer]
Appwrite has the same limits as the under-the-hood database. Currently we support both MariaDB and MySQL, and soon MongoDB. I'm not aware of any such limit to any of them. We've also seen users run in similar scale before.

**Reply by issasaleh** [reply]
MongoDB, that's wonderful news.
